bower.json

```diff
@@ -27,6 +27,7 @@
     "select2": "~4.0.0",
     "sticky": "latest",
     "store.js": "latest",
+    "susy" : "latest",
```

Comment:
I know other's have 'latest', but new deps should be locked to a version (using `--save-dev`)